bio/tmb/pyeffgenomesize/wrapper.py

```
@@ -0,0 +1,49 @@
+from snakemake.shell import shell
+from tempfile import TemporaryDirectory
+
+log = snakemake.log_fmt_shell(stdout=False, stderr=True)
+extra = snakemake.params.get("extra", "")
+
+# Optional IO files
+bam = snakemake.input.get("bam")
+if bam:
+    extra += f" --bam '{bam}' --mosdepth "
+
+thresholds = snakemake.output.get("thresholds")
```

Ruff (0.12.2)

Undefined name `snakemake`

(F821)